src/Internal/Logger.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Class Logger
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal;
+
+use WC_Logger;
+use WC_Payments;
+use WCPay\Core\Mode;
+use WCPay\Internal\Proxy\LegacyProxy;
```

Comment:
The `LegacyProxy` is not needed in this class.

Reply:
Removed